solr/config/synonyms.txt

```diff
@@ -0,0 +1,208 @@
+Alzheimer diagnosis and management => Alzheimer disease diagnosis and management
+Coronary artery disease => Coronary heart disease
+
+# Thesaurus 2016-2 PLT-182
+Transsexuals => Transgender people
+Sexual preferences => Sexuality groupings
+Neurorehabilitation and trauma => Neurorehabilitation
+End of life => End of life care
+
+# Thesaurus 2017-1 RARO-225
+Fishes => Fish
+Open access => Open access publishing
+Spring => Spring (season)
+Pleura => Pleurae
+Ice cap => Ice caps
+Ice sheet => Ice sheets
+Ice shelf => Ice shelves
+Ice core => Ice cores
+Hydrocortisone => Cortisol
+National governments other than U.S. => National governments
+State government => State governments
+City government => City governments
+Local government => Local governments
+Municipal government => Municipal governments
+Functional connectivity MRI => Functional connectivity magnetic resonance imaging
+Resting state fMRI => Resting state functional magnetic resonance imaging
+Deletions => Chromosomal deletions
+Duplications => Chromosomal duplications
+Inversions => Chromosomal inversions
+Translocations => Chromosomal translocations
+Zinc finger domain => Zinc finger domains
+SH2 domain => SH2 domains
+Basic leucine zipper domain => Basic leucine zipper domains
+Basic helix-loop-helix domain => Basic helix-loop-helix domains
+Ankyrin repeat domain => Ankyrin repeat domains
+Alzheimer disease => Alzheimer's disease
+Familial Alzheimer disease => Familial Alzheimer's disease
+Sporadic Alzheimer disease => Sporadic Alzheimer's disease
+Alzheimer disease diagnosis and management => Alzheimer's disease diagnosis and management
+African Americans => African American people
+Africans => African people
+Albanians => Albanian people
+Arabs => Arabic people
+Armenians => Armenian people
+Batak => Batak people
+Belarusians => Belarusian people
+Bulgarians => Bulgarian people
+Croats => Croatian people
+Danes => Danish people
+Estonians => Estonian people
+Ethnic Malays => Malay people
+Fijians => Fijian people
+Finns => Finnish people
+Han Chinese => Han Chinese people
+Hispanic Americans => Hispanic American people
+Indigenous Australians => Indigenous Australian people
+Jews => Jewish people
+Latin Americans => Latin American people
+Latvians => Latvian people
+Lithuanians => Lithuanian people
+Montenegrins => Montenegrin people
+Native Americans => Native American people
+Polynesians => Polynesian people
+Quechuas => Quechua people
+Romanians => Romanian people
+Serbs => Serbian people
+Slavs => Slavic people
+Slovaks => Slovakian people
+Slovenes => Slovenian people
+Somalis => Somalian people
+Swedes => Swedish people
+Bugis => Buginese people
+Oriyas => Odishan people
+Sasak => Sasak people
+Pardo Brazilians => Pardo Brazilian people
+Obsessive�compulsive personality disorder => Obsessive-compulsive personality disorder
+Georgia (United States) => Georgia (United States state)
+Linguistic morphology => Morphology (linguistics)
+Archaeans => Archaea
```

Comment:
the dev and stage journals collections actually use a synonyms.txt that's slightly expanded from the current prod one. See: http://solr-mega-dev.soma.plos.org/solr/#/journals_stage/files?file=synonyms.txt

Note 

The reason for this is because the dev and stage synonyms.txt contain entries for the 2017-2 thesaurus release, but 2017-2 was never deployed on prod (intentionally anyways, the 2017-2 snuck its way into prod which cause this bug: https://jira.plos.org/jira/browse/AMBR-1085).